### PR TITLE
Update welcome message to include validators.

### PR DESCRIPTION
**Pull request by matthewcarlreetz**

### CODE REVIEW

File: src/features/hotspots/root/WelcomeOverview.tsx [hunk 1]
```diff
@@ -38,19 +41,33 @@ const TimeOfDayTitle = ({ date }: { date: Date }) => {
...
+  )
+
+  const loading = useMemo(() => accountRewards.loading || validatorsLoading, [
     accountRewards.loading,
+    validatorsLoading,
   ])
 
+  useVisible({
+    onAppear: () => {
+      dispatch(fetchValidators())
```
Comment (tyler-whitman) [inline src/features/hotspots/root/WelcomeOverview.tsx:67]: Should we move this to hotspots view where the `fetchAccountRewards()` happens? Or maybe move that here?

File: src/utils/appDataClient.ts [hunk 2]
```diff
@@ -52,6 +52,15 @@ export const getHotspots = async () => {
   return newHotspotList.takeJSON(1000)
 }
 
+export const getValidators = async () => {
+  Logger.breadcrumb('getValidators', breadcrumbOpts)
+  const address = await getAddress()
+  if (!address) return []
+
+  const newValidatorsList = await client.account(address).validators.list()
+  return newValidatorsList.takeJSON(1000)
```
Comment (tyler-whitman) [inline src/utils/appDataClient.ts:61]: Does this api call have a cursor? If not maybe we should use the `MAX` variable instead to make it more clear?

**Review by tyler-whitman — APPROVED** [review]
Looks good, tested it with a no validator but hotspot account and an only validator account. Don't have one with both to try.

File: src/features/hotspots/root/WelcomeOverview.tsx (hunk 1 above)
Comment (matthewcarlreetz) [reply src/features/hotspots/root/WelcomeOverview.tsx]: yes, good call

File: src/utils/appDataClient.ts (hunk 2 above)
Comment (matthewcarlreetz) [reply src/utils/appDataClient.ts]: I changed it to max and a few others that should be using too.

**Pull request merged**